helper-cli/src/main/kotlin/commands/GetPackageLicensesCommand.kt

```diff
@@ -121,7 +122,7 @@ class GetPackageLicensesCommand : CliktCommand(
 }
 
 private fun Collection<LicenseFinding>.toSpdxExpression(): String =
-    asSequence().distinct().sorted().joinToString(" AND ")
+    asSequence().distinct().joinToString(" AND ").toSpdx().sort().toString()
```

Comment:
You are now converting the `LicenseFinding`s to strings and still not use `reduce()`:
```
asSequence().map { it.license }.distinct().reduce(SpdxEpxression::and).sort().toString()
```

Reply:
I've added the reduce in a separate commit now.